src/components/TimeSeries/TimeSeriesForm.tsx

```diff
@@ -0,0 +1,90 @@
+// TODO: Fix eslint errors
+
+/* eslint-disable @typescript-eslint/no-misused-promises */
+import { format, subDays } from 'date-fns';
+import { useEffect, useState } from 'react';
+import { useForm } from 'react-hook-form';
+
+import { stations } from '../../mocks/timeseries/stations.json';
+import timeseries from '../../mocks/timeseries/timeseries.json';
+import {
+  type Station,
+  type TimeSeriesResponse
+} from '../../types/models/timeSeries';
```

Comment:
```suggestion
import { stations } from '@/mocks/timeseries/stations.json';
import timeseries from '@/mocks/timeseries/timeseries.json';
import {
  type Station,
  type TimeSeriesResponse
} from '@/types/models/timeSeries';
```
For deeply nested files use prefix imports using `@/` to create import relative to the `src` directory. It will result in a more readable and shorter path.